consensus.go

```diff
@@ -0,0 +1,224 @@
+package dcrlibwallet
+
+import (
+	"fmt"
+	"sort"
+
+	"decred.org/dcrwallet/v2/errors"
+	w "decred.org/dcrwallet/v2/wallet"
+
+	"github.com/decred/dcrd/chaincfg/chainhash"
+	"github.com/decred/dcrd/chaincfg/v3"
+	"github.com/decred/dcrd/wire"
+)
+
+const (
+	dcrdataAgendasAPIMainnetUrl = "https://dcrdata.decred.org/api/agendas"
+	dcrdataAgendasAPITestnetUrl = "https://testnet.decred.org/api/agendas"
+
+	// AgendaStatusUpcoming used to define an agenda yet to vote,
+	// called from godcr.
+	AgendaStatusUpcoming = "upcoming"
+
+	// AgendaStatusInProgress used to define an agenda with voting ongoing,
+	// called from godcr.
+	AgendaStatusInProgress = "in progress"
+
+	// AgendaStatusFinished used to define an agenda that has finished voting,
+	// called from godcr.
+	AgendaStatusFinished = "finished"
+)
```

Comment:
No need to add `called from godcr`. Other consumers of this public variables may find that confusing.